classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(lebal)
+    @lebal = lebal
+    @student = []
```

Comment:
- [OPTIONAL] This is optional but I highly recommend you write this in the plural form👉🏼 i.e. `students`. Why? It sounds right since this will be a collection of students.